src/components/image/index.tsx

```diff
@@ -20,29 +20,30 @@
  */
 
 import React, { ReactElement, useState, useEffect, useContext } from "react";
-import { Thing, Url, UrlString, getUrl } from "@inrupt/solid-client";
-import { overwriteFile, retrieveFile } from "../../helpers";
+
+import {
+  overwriteFile,
+  retrieveFile,
+  CommonProperties,
+  useProperty,
+} from "../../helpers";
+
 import { SessionContext } from "../../context/sessionContext";
-import ThingContext from "../../context/thingContext";
 
 export type Props = {
-  thing?: Thing;
-  property: Url | UrlString;
-  edit?: boolean;
-  autosave?: boolean;
   maxSize?: number;
   inputProps?: React.InputHTMLAttributes<HTMLInputElement>;
-  onSave?: () => void;
-  onError?: (error: Error) => void;
   errorComponent?: React.ComponentType<{ error: Error }>;
-} & React.ImgHTMLAttributes<HTMLImageElement>;
+} & CommonProperties &
+  React.ImgHTMLAttributes<HTMLImageElement>;
 
 /**
  * Fetches and displays an image, from a src found at a given property of a given [Thing](https://docs.inrupt.com/developer-tools/javascript/client-libraries/reference/glossary/#term-Thing). Can also be used to upload a new/replacement image file.
```

Comment:
I (personally) set the ruler in my IDE at 80 chars, and always truncate comments at that, although I know **_our_** Prettier setup sets it at 100 chars (and the out-of-the-box Prettier recommendation is 80 chars), but I do find sticking to that 80 char limit really helps PR reviews (and especially so if reviewing on a phone!), so this particular comment really tends to 'stick out' regardless of how massive one's monitor is :) !

Reply:
Agreed, weird that the linter isn't complaining. We can investigate and update the linter seperately.